src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntry.cmp

```diff
@@ -44,6 +44,7 @@
     <aura:attribute name="data" type="List" default="[]" description="Data Import records that appear in the table" access="public"/>
     <aura:attribute name="errors" type="Map" description="Dry run errors to display on the lightning:dataTable" access="public"/>
     <aura:attribute name="columns" type="List" description="Columns for the table" access="public"/>
+    <aura:attribute name="referenceFields" type="List" description="Reference fields used in this batch" access="public"/>
```

Comment:
Does this need to be public? (Does 'columns' for that matter?)

Reply:
probably not!